crates/primitives/src/result.rs

```diff
@@ -180,7 +180,7 @@ pub enum InvalidTransaction {
     RejectCallerWithCode,
     /// Transaction account does not have enough amount of ether to cover transferred value and gas_limit*gas_price.
     LackOfFundForMaxFee {
-        fee: u64,
+        fee: U256,
```

Comment:
Could you make them as `Box<U256>` it will reduce the size of the type, and balance too?